src/test/java/Homework19.java

```diff
@@ -0,0 +1,62 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework19 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        //      Added ChromeOptions argument below to fix websocket error
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+
+         // WHEN
+        // Performing actions: login, select playlist, delete playlist,verify deleted message.
+        login("india.messam@testpro.io", "slcTalgy");
+        Thread.sleep(2000);
+        choosePlaylist("First user playlist");
+        driver.quit();
+    }
+
+
+    // Method to choose a playlist by name.
+    public void choosePlaylist(String FirstUserPlaylist) throws InterruptedException {
+        // Locating and clicking the playlist with the specified name.
+        WebElement playlist = driver.findElement(By.cssSelector("[href='#!/playlist/95502']"));
+        playlist.click();
+        Thread.sleep(2000);
+
+    }
+    public void clickDeletePlaylistBtn(String DeletePlaylist) throws InterruptedException{
+        WebElement deleteBtn = driver.findElement(By.cssSelector("[class='del btn-delete-playlist']"));
+        deleteBtn.click();
+        Thread.sleep(2000);
+    }
+    // Method to get the success message after adding to the playlist.
+    public String getDeletePlaylistSuccessMsg() {
+        // Locating and retrieving the success message.
+        WebElement notification = driver.findElement(By.cssSelector("div.success.show"));
+        return notification.getText();
+    }
+
+
+    }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._